vaadin-portlet/src/test/java/com/vaadin/flow/portal/PortletStreamResourceRegistryTest.java

```diff
@@ -53,37 +52,6 @@ public void checkHasLock() {
         registry = new PortletStreamResourceRegistry(session);
     }
 
-    @Test
-    public void getResourceUri_mimeContent_returnsEmbeddedUrl() {
```

Comment:
May I ask why this test is no more needed?

Reply:
That was supposed to be returned as the mime change is actually needed for the event to be targeted correctly. I'll add it back